File: 02-Tech-Module/01-Programming-Fundamentals/06-Data-Types-And-Variables-Exercises/13-Vowel-Or-Digit/VowelOrDigit.cs
using System;

class VowelOrDigit
{
    static void Main()
    {
        char symbol = char.Parse(Console.ReadLine());

        if (symbol == 'a' || symbol == 'e' || symbol == 'o' || symbol == 'i' || symbol == 'u')
        {
            Console.WriteLine("vowel");
        }
        else if (symbol >= 48 && symbol <= 57)
        {
            Console.WriteLine("digit");
        }
        else
        {
            Console.WriteLine("other");
        }
    }
}

File: 02-Tech-Module/02-Software-Technologies/12-Java-Basics-Exercises/src/VowelOrDigit.java
import java.util.Scanner;

public class VowelOrDigit {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String input = scanner.nextLine();

        if (tryParseInt(input)) {
            System.out.printf("digit");
        }else{
            switch (input){
                case "a":
                    System.out.printf("vowel");
                    break;
                case "e":
                    System.out.printf("vowel");
                    break;
                case "i":
                    System.out.printf("vowel");
                    break;
                case "o":
                    System.out.printf("vowel");
                    break;
                case "u":
                    System.out.printf("vowel");
                    break;
                    default:
                        System.out.println("other");
                        break;
            }
        }

    }

    static boolean tryParseInt(String value) {
        try {
            Integer.parseInt(value);
            return true;
        } catch (NumberFormatException e) {
            return false;
        }
    }
}


